internal/config.go

```diff
@@ -60,8 +60,7 @@ type TelegramSettings struct {
 
 // ImgurSettings includes settings related to Imgur uploading for Telegram photos
 type ImgurSettings struct {
-	UseImgurForImageLinks bool   `env:"USE_IMGUR_FOR_IMAGE" envDefault:"true"`
-	ImgurClientID         string `env:"IMGUR_CLIENT_ID" envDefault:"7d6b00b87043f58"`
+	ImgurClientID string `env:"IMGUR_CLIENT_ID" envDefault:"7d6b00b87043f58"`
```

Comment:
Is this a new API token? I assume we can distribute this as a TeleIRC-default while encouraging others to rotate with their own API token.

Reply:
This line just gets rid of the `UseImgurForImageLinks` line. We already use a default `ImgurClientID`.